src/Uno.UI.RuntimeTests/Tests/Windows_Storage/Given_DataWriterReader.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Resources;
+using Windows.ApplicationModel.Resources.Core;
+using Windows.Storage.Streams;
+
+namespace Uno.UI.RuntimeTests.Tests
+{
+	[TestClass]
+	public class Given_DataWriterReader
+	{
+		[TestMethod]
+		public async Task When_WriteThenRead()
+		{
+			bool saveBool = true;
+			byte saveByte = 10;
+			int saveInt = 12345;
+			double saveDouble = 12345.678;
+			DateTimeOffset saveDate = DateTimeOffset.Now;
+			TimeSpan saveTimeSpan = TimeSpan.FromMinutes(10);
+			Guid saveGuid = Guid.NewGuid();
+
+			bool error = false;
+			var writer = new DataWriter();
+			try
+			{
+				writer.WriteBoolean(saveBool);
+				writer.WriteByte(saveByte);
+				writer.WriteInt32(saveInt);
+				writer.WriteDouble(saveDouble);
+				writer.WriteDateTime(saveDate);
+				writer.WriteTimeSpan(saveTimeSpan);
+				writer.WriteGuid(saveGuid);
+			}
+			catch
+			{
+				error = true; 
+			}
+
+			if(error)
+			{
+				Assert.Fail("FAIL while writing (DataWriter)");
+			}
+
+			var buffer = writer.DetachBuffer();
+
+			var reader = DataReader.FromBuffer(buffer);
+
+			// variables has to be initialized, because if not, we get CS0165 Use of unassigned local variable 
+			bool loadBool =false;
+			byte loadByte=0;
+			int loadInt=0;
+			double loadDouble=0;
+			DateTimeOffset loadDate = new DateTimeOffset(2000, 1, 1, 0, 0, 0, TimeSpan.FromSeconds(10));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'loadDate'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)